### DISCUSSION TITLE
Seeed Studio XIAO nRF52840 Battery Charge Chip どっちがほんと？？

### DISCUSSION THREAD

**Discussion by Mihokun** [algyan/XIAOGYAN#67] [General]
seeed wiki では BQ25101 と書いてる（サイト内 表中）けど
回路図では BQ25100 と書かれてる（サイト内 ResourcesからDL）

どっちがほんと？？

URL：https://wiki.seeedstudio.com/XIAO_BLE/

XIAO BLE(XIAO nRF52840) と DTP401525(PHR) 110mAh
リチウムイオンポリマー電池 3.7V 110mAh を買った。
XIAO BLE は バッテリーチャージャーが搭載！ということなので買ってみたんだけど、
「充電しすぎにならないか？」
「充電終わったら 緑LED消えてくれたりするのかな？」
など、すごく不安。。。

**Comment by matsujirushi** [comment]
ForumもしくはDiscordへ聞いてみては？
https://wiki.seeedstudio.com/XIAO_BLE/#tech-support--product-discussion

**Reply by matsujirushi** [reply]
このDiscussionで聞くほうが良いかも。（Wikiに対するDiscussionなので）
https://github.com/Seeed-Studio/wiki-documents/discussions/122

**Reply by matsujirushi** [reply]
書きました。
https://github.com/Seeed-Studio/wiki-documents/discussions/122#discussioncomment-6310563

**Reply by matsujirushi** [reply]
BQ25101でした。

https://github.com/Seeed-Studio/wiki-documents/discussions/122#discussioncomment-6312440
